src/Orleans/Core/GrainAttributes.cs

```diff
@@ -86,6 +87,37 @@ public sealed class ImmutableAttribute : Attribute
         }
     }
 
+    namespace MultiCluster
+    {
+        /// <summary>
+        /// base class for multi cluster registration strategies.
+        /// </summary>
+        public abstract class RegistrationAttribute : Attribute
+        {
+            internal MultiClusterRegistrationStrategy RegistrationStrategy { get; private set; }
+
+            internal RegistrationAttribute(MultiClusterRegistrationStrategy strategy)
+            {
+                RegistrationStrategy = strategy ?? MultiClusterRegistrationStrategy.GetDefault();
+            }
+        }
+
+        /// <summary>
+        /// This attribute indicates that instances of the marked grain class
+        /// will have an independent instance for each cluster with 
+        /// no coordination. This is the default, so no need to explicitly specify this attributes 
```

Comment:
If we are in agreement that global single instance will be the default, then let's remove this contradictory  statement here now, so that we don't need to remember to fix it later.